lib/AST/Bridging/ASTContextBridging.cpp

```diff
@@ -167,3 +168,9 @@ bool BridgedASTContext_canImport(BridgedASTContext cContext,
       builder.get(), canImportLoc.unbridged(), version,
       versionKind == CanImportUnderlyingVersion);
 }
+
+BridgedAvailabilityMacroMap
+BridgedASTContext_getAvailabilityMacroMap(BridgedASTContext cContext) {
+  return const_cast<AvailabilityMacroMap *>(
```

Comment:
Would be nice if `BridgedAvailabilityMacroMap` could store a const pointer, maybe we should define it manually instead of using `AST_BRIDGING_WRAPPER`? Or maybe we could add a nullable variant of that macro